web/cypress/support/commands.js

```diff
@@ -72,6 +72,17 @@ Cypress.Commands.add('configurePlugins', (cases) => {
           });
         });
 
+        cy.get(domSelector.codeMirrorMode).invoke('text').then(text => {
+          if (text === 'Form') {
+            // FIXME: https://github.com/cypress-io/cypress/issues/7306
```

Comment:
please create a issue for it

Reply:
> please create a issue for it

ok, https://github.com/apache/apisix-dashboard/issues/1738